Frontend/ui-customer/src/Flow.purs

```diff
@@ -1197,11 +1219,28 @@ homeScreenFlow = do
         homeScreenFlow
       else do
         PlaceName placeDetails <- getPlaceName lat lon gateAddress
+        let currentLocationItem = Just (dummyLocationListState{
+                              prefixImageUrl = "ny_ic_recent_search,https://assets.juspay.in/nammayatri/images/user/ny_ic_recent_search.png"
+                            , postfixImageUrl = "ny_ic_fav,https://assets.juspay.in/nammayatri/images/user/ny_ic_fav.png"
+                            , title = (fromMaybe "" ((split (Pattern ",") (placeDetails.formattedAddress)) !! 0))
+                            , subTitle = (drop ((fromMaybe 0 (indexOf (Pattern ",") (placeDetails.formattedAddress))) + 2) (placeDetails.formattedAddress))
```

Comment:
try to make helper functions for such cases and write it like this -

`subTitle = do`
  `formattedAddress <- placeDetails.formattedAddress`
 ` let index = fromMaybe 0 (indexOf (Pattern ",") formattedAddress)`
 ` pure $ drop (index + 2) formattedAddress`

Reply:
it's simplified and easy to understand, having logic in one line makes it difficult to understand and review the code